doc/source/development.rst

```diff
@@ -193,143 +201,242 @@ Make and commit your changes.
-method. To make it easier to follow this method, :command:`bumpversion` is
-used as a tool.
+Bumping the Version
+~~~~~~~~~~~~~~~~~~~
+
+The KIWI project follows the `Semantic Versioning <https://semver.org>`_
+scheme. We use the :command:`bumpversion` tool for consistent versioning.
 
-Follow these instructions to raise the major, minor, or patch part of a
-version:
+Follow these instructions to bump the major, minor, or patch part of the
+KIWI version:
 
 *  For backwards-compatible bug fixes:
 
-   .. code:: bash
+   .. code-block:: shell-session
 
     $ bumpversion patch
 
 *  For additional functionality in a backwards-compatible manner. When
-   changed, the patch level is set back to zero:
+   changed, the patch level is reset to zero:
 
-   .. code:: bash
+   .. code-block:: shell-session
 
     $ bumpversion minor
 
 *  For incompatible API changes. When changed, the patch and minor
-   levels are set back to zero:
+   levels are reset to zero:
 
-   .. code:: bash
+   .. code-block:: shell-session
 
     $ bumpversion major
 
-Creating a Package
-~~~~~~~~~~~~~~~~~~
-
-The creation of RPM package sources has to be done by calling the
-following make target:
+Creating a RPM Package
+~~~~~~~~~~~~~~~~~~~~~~
 
-.. code:: bash
+We provide a template for a RPM spec file in
+:file:`package/python-kiwi-spec-template` alongside with a rpmlint
+configuration file and an automatically updated
+:file:`python-kiwi.changes`.
 
-   $ make build
+To create the necessary files to build a RPM package via `rpmbuild`, run:
 
-The sources are collected below the :file:`dist/` directory. In there you
-will find all required files to submit a package to the Open Build
-Service or just build it with :command:`rpmbuild`.
+.. code-block:: shell-session
 
-Building Documentation
-~~~~~~~~~~~~~~~~~~~~~~
+   $ make build
 
-The documentation is implemented using Sphinx with the ReST markup. In
-order to build the documentation just call:
+The sources are collected in the :file:`dist/` directory. These can be
+directly build it with :command:`rpmbuild`, :command:`fedpkg` or submitted
+to the Open Build Service using :command:`osc`.
 
-.. code:: bash
 
-    tox -e doc
+.. [#f1] Sphinx requires a plethora of additional LaTeX
+         packages. Unfortunately there is currently no comprehensive list
+         available. On Ubuntu/Debian installing `texlive-latex-extra`
+         should be sufficient, for Fedora please consult the package list
+         from :file:`.gitlab-ci.yml`.
 
-Whenever a change in the documentation is pushed to GitHub, it will be
-automatically updated via :command:`travis-sphinx` and is available at:
+.. [#f2] Configure your editor to automatically break lines and/or reformat
+         paragraphs. For Emacs you can use `M-x set-fill-column RET 75` and
+         `M-x auto-fill-mode RET`. For editing reStructuredText we
+         recommend `rst-mode` (built-in to Emacs since version `23.1`).
```

Comment:
Do you have also recommendations for vi users? :wink:

Reply:
Not really, I've left the Cult of VIM a while ago and converted to the Church of Emacs. St. iGNUcius does not allow to have other deities beside him, so I must abstain from recommending anything to vi users. `</joking>`

Jokes aside: not really, I've never used vi properly. I'll try to dig something up, but a recommendation from an actual vi user is probably more helpful :wink: